### PR TITLE
net-misc/rabbitmq-server: bump to 3.8.0.

### PR DESCRIPTION
**Pull request by dargor**
Signed-off-by: Gabriel Linder <linder.gabriel@gmail.com>
Package-Manager: Portage-2.3.76, Repoman-2.3.17

### CODE REVIEW

File: net-misc/rabbitmq-server/files/rabbitmq-server.init-r4 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	need net
+	use dns epmd
+}
+
+start() {
+	ebegin "Starting RabbitMQ"
+	/sbin/start-stop-daemon \
```
Comment (arkamar) [inline net-misc/rabbitmq-server/files/rabbitmq-server.init-r4]: Ebuild creates user and group `rabbitmq`. I would suggest to use them.
```
--user rabbitmq --group rabbitmq
```

Comment (dargor) [reply net-misc/rabbitmq-server/files/rabbitmq-server.init-r4]: User switching is done by `files/rabbitmq-script-wrapper`.

Comment (arkamar) [reply net-misc/rabbitmq-server/files/rabbitmq-server.init-r4]: Oh, I see, sorry for the noise.

**Comment by ultrabug** [comment]
Thanks!

**Pull request closed without merging**